Javascript/translator.js

```diff
@@ -0,0 +1,43 @@
+import mapping from "./mapping.js";
+
+//Text to morse convertor
+export const textToMorse = () => {
+  var inputValue = document
+    .getElementById("textarea__input--english")
+    .value.toUpperCase();
+  let splittedInputValue = inputValue.split("");
+  let translatedToMorse = splittedInputValue.map((character) => {
+    if (!mapping[character]) {
+      return "🤷";
+    } else {
+      return mapping[character];
+    }
+  });
+  document.getElementById("textarea__input--morse").value =
+    translatedToMorse.join(" ");
```

Comment:
This will need to be split into more functions. 
One of them will need to be a pure function that takes in a string of text and returns a string of morse code 
Another one will deal with updating the DOM

Reply:
Got it. Completed the test cases